mw-webapp/src/logic/wayPage/wayStatistics/statisticWidget/StatisticWidget.tsx

```diff
@@ -29,7 +30,7 @@ interface StatisticLabelsProps {
 export const StatisticWidget = (props: PropsWithChildren<StatisticLabelsProps>) => {
   return (
     <VerticalContainer className={styles.statisticBlock}>
-      <HorizontalContainer className={styles.titleCheckbox}>
+      <HorizontalContainer className={clsx(styles.titleCheckbox, props.isEditable ? styles.titleCheckboxIndex : "")}>
```

Comment:
```suggestion
      <HorizontalContainer className={clsx(styles.titleCheckbox, props.isEditable && styles.titleCheckboxIndex)}>
```

Reply:
I used the same logic like 
![image](https://github.com/tritonJS826/masters-way/assets/112805668/d4ea7b89-4bd3-4436-8a06-cc003179e229)
because props.isEditable === isCheckboxShown
![image](https://github.com/tritonJS826/masters-way/assets/112805668/6e4d2242-071c-4feb-9c37-d4f3b9d3e15f)
if we have checkbox in StatisticWidget(isCheckboxShown) =>  container titleCheckbox need add z-index to up under barChartContainer with canwas. if remove this check 
![image](https://github.com/tritonJS826/masters-way/assets/112805668/fbc1cd07-c241-4be1-b900-aa4c1cbf597e)
we have titleCheckboxcontainer in all StatisticWidget, but we need it only with isCheckboxShown

Reply:
withot this check 
![image](https://github.com/tritonJS826/masters-way/assets/112805668/255cdfe6-0ea5-49f9-b24d-68bba08b78c8)

Reply:
suggest another way to fix this bug? let's disscus in conversation

Reply:
its not good resolve because we have checkbox in StatisticWidget only in Modal window, if we add to another component isCheckboxShown it will be problem

Reply:
maybe this problem will be resolved if you add z-index to checkbox,  not to wrapper titleCheckbox?
Let's try)

Reply:
fixed
it's doesn't work(z-index) without add position:relative in titleCheckbox. With relative everything allright